pages/index.tsx

```diff
@@ -7,16 +7,24 @@ import Values from "@/components/general/OurValues";
 import FeaturedWork from "../components/general/FeaturedWork";
 import LatestNews from "@/components/general/LatestNews";
 import HowWework from "@/components/general/HowWeWork";
+import EViewPortQuery from "@/constants/viewPortSize";
+import useMediaQuery from "@/hooks/useMediaQuery";
+
+const { DESKTOP, TABLET, PHONE } = EViewPortQuery;
 
 export default function Home() {
+  let isDesktopSize = useMediaQuery(DESKTOP);
+  let isTabletSize = useMediaQuery(TABLET);
+  let isPhoneSize = useMediaQuery(PHONE);
+
   return (
     <>
       <Head>
         <title>Taodemy Website</title>
         <meta name="description" content="Generated by Taodemy" />
       </Head>
       <main>
-        <Hero />
+        <Hero isPhoneSize={isPhoneSize} />
```

Comment:
Sorry, I don't think it is good to put the isPhoneSize in index page, because it decrease the cleanliness and readability of the index page.

Reply:
It is indeed a bit cumbersome to repeatedly import the useMediaQuery hook（I imported twice in my last PR）, perhaps we can discuss it in the standing meeting tonight.

Reply:
@yunshen513 @Charles-Xing That is OK, let's have a discuss in tonight meeting.

Reply:
I think it is a very good idea to put it in index page.